kernel/fde/cmd_helper.go

```diff
@@ -49,12 +52,34 @@ var fdeInitramfsHelperPollWaitParanoiaFactor = 2
 // overridden in tests
 var fdeInitramfsHelperCommandExtra []string
 
+// Wait for unit not being active
```

Comment:
Maybe a short comment here *why* we do this? Like: `// Wait for the unit not being active as otherwise the dir may be cleaned up too early and that causes systemd to hang because ...` ?

Reply:
Done.